src/lib/profile/scope.cpp

```diff
@@ -57,6 +57,7 @@
 #include <iomanip>
 #include <stdexcept>
 #include <sstream>
+#include <iomanip>
```

Comment:
Unrelated, remove. If there's a compile failure in this file because of it let me know.

Reply:
@blue42u Yes, there's a compilation error `setw is not a member of std after taking it out` after taking out `#include<iomainip>`

Reply:
gcc 8.5.0 was used

Reply:
@Jokeren I just tried in Spack, `hpctoolkit@develop %gcc@8.5.0` compiles just fine with the Spack-built GCC. I only just noticed, but there's literally an `#include <iomanip>` about 3 lines above your addition.

So remove.